core/src/main/java/org/apache/brooklyn/core/BrooklynFeatureEnablement.java

```diff
@@ -61,6 +61,37 @@
     /** whether feeds are automatically registered when set on entities, so that they are persisted */
     public static final String FEATURE_FEED_REGISTRATION_PROPERTY = FEATURE_PROPERTY_PREFIX+".feedRegistration";
 
+    /**
```

Comment:
I didn't say that CORS was _always_ insecure; used judiciously, it can be a perfectly acceptable mechanism for many use-cases.  Used badly or inappropriately, it can open you up to increased security risks, but the same can be said about lots of things.  I would remove the warnings from this comment, and simply explain what the feature is that is being enabled.  It is up to users to determine whether CORS is applicable in their case and to enable it if so.